gateway/src/main/kotlin/DefaultGateway.kt

```diff
@@ -159,7 +159,7 @@ public class DefaultGateway(private val data: DefaultGatewayData) : Gateway {
 
 
     private suspend fun readSocket() {
-        socket.incoming.asFlow().buffer(Channel.UNLIMITED).collect {
+        socket.incoming.asFlow().buffer(Channel.UNLIMITED).collectLatest {
```

Comment:
I remember this was due to a bug in previous version of coroutine lib, should this be changed back?

Reply:
yes, it should, `collectLatest` does a different thing

Reply:
I remember they marked this as internal and then not made it internal again